api-doclet/build.gradle

```diff
@@ -1,6 +1,8 @@
 description = 'Conscrypt: API Doclet'
 
-dependencies {
-    // This should be removed when upgrading to Gradle 5.x
-    compile files(org.gradle.internal.jvm.Jvm.current().toolsJar)
+def toolsJar = org.gradle.internal.jvm.Jvm.current().toolsJar
+if (toolsJar != null) {
```

Comment:
Nice! This has been bugging me on jdk > 9 👍

Reply:
I'm not sure this will work for long. The doclet needs to be rewritten for >=JDK9 and <=JDK8.